app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -26,23 +26,25 @@ export default function DeleteAlert({ refetch, onClose, selectedNoteIds }) {
       showFooter
       submitButtonProps={{
         style: "danger",
-        label: "Continue anyway",
+        label: "Delete",
         loading: deleting,
         onClick: handleDelete
       }}
       onClose={onClose}
     >
       <div className="flex">
         <div className="flex items-center justify-center flex-shrink-0 w-10 h-10 bg-red-100 rounded-full">
-          <i className="text-red-500 ri-alarm-warning-fill ri-lg"></i>
+          <i className="text-red-500 ri-alarm-warning-line ri-lg"></i>
         </div>
 
         <div className="ml-4">
           <h3 className="mb-2 text-lg font-medium text-gray-700">
-            Delete {selectedNoteIds.length} notes?
+            Delete Note
           </h3>
           <div className="text-sm leading-5 text-gray-500">
-            Are you sure you want to continue? This cannot be undone.
+            Are you sure you want to delete the note? All of your data will be
+            permanently removed from our database forever. This action cannot be
+            undone.
           </div>
         </div>
       </div>
```

Comment:
Please replace this component and use [neetoUI Alert Component](https://neetoui.netlify.app/#/alert).